templates/katello.yml.erb

```diff
@@ -12,43 +12,24 @@
 # see /usr/share/katello/config.katello_defaults.yml for available options
 
 common:
-  app_mode: 'katello'
   <%- if @cdn_ssl_version && !@cdn_ssl_version.strip.empty?-%>
   cdn_ssl_version: <%= @cdn_ssl_version %>
   <%- end -%>
 
-  warden:  <%= scope.lookupvar("katello::params::auth_method") %>
-  ldap_roles: <%= scope.lookupvar("katello::params::ldap_roles") %>
-  validate_ldap: <%= scope.lookupvar("katello::params::validate_ldap") %>
   rest_client_timeout: 120
 
-  url_prefix: <%= @deployment_url %>
-  host: <%= has_variable?("fqdn") ? @fqdn : hostname %>
-  port:
-  use_ssl: true
-  use_foreman: <%= scope.lookupvar("katello::params::use_foreman") %>
-
-  post_sync_url: https://localhost<%= scope.lookupvar("katello::params::deployment_url") %>/api/v2/repositories/sync_complete?token=<%= scope.lookupvar("post_sync_token") %>
+  post_sync_url: https://localhost<%= @deployment_url %>/api/v2/repositories/sync_complete?token=<%= @post_sync_token %>
 
   candlepin:
-    url: <%= scope.lookupvar("katello::params::candlepin_url") %>
+    url: <%= @candlepin_url %>
```

Comment:
https://docs.puppetlabs.com/guides/templating.html#referencing-variables mentions that they should be in scope and since the template is rendered from ```katello::config```, I'd *expect* everything from the ```katello``` scope to be unavailable.